app/assets/javascripts/dashboard.js

```diff
@@ -28,14 +28,13 @@ MinionPoller = {
         }
         $(".nodes-container tbody").html(rendered);
 
-        var unassignedMinions = data.unassigned_minions;
+        var unassigned = data.unassigned_minions;
 
-        if (unassignedMinions.length > 0) {
-          $('#unassigned_count').html(unassignedMinions.length + " \
-            <strong>new</strong> nodes are available but have not been added to the cluster yet"
-          );
-        } else {
+        if (!unassigned || unassigned.length === 0) {
           $('#unassigned_count').html('');
+        } else {
+          $('#unassigned_count').html(data.unassigned_minions.length + " \
+            <strong>new</strong> nodes are available but have not been added to the cluster yet");
```

Comment:
In the (near?) future they will be able to add them so I don't think its too important to change the message for now. In any case, if we change it, it should also be changed [here](https://github.com/kubic-project/velum/blob/master/app/views/dashboard/index.html.slim#L8) (reminder).